File: output/ChatGPT/codenet/Java/C++/s380356166.cpp
#include <iostream>
using namespace std;

int main() {
    int ans = 0;
    int K, S, wk;
    cin >> K >> S;
    for(int x=0; x <= K; x++) {
        for(int y=0; y <= K; y++) {
            wk = S - x - y;
            if(wk >= 0 && wk <= K) {
                ans++;
            }
        }
    }
    cout << ans << endl;

    return 0;
}

File: output/ChatGPT/codenet/Java/Python/s380356166.py
def solve(K, S):
    ans = 0
    for x in range(K+1):
        for y in range(K+1):
            wk = S - x - y
            if 0 <= wk <= K:
                ans += 1
    return ans

K = int(input())
S = int(input())
print(solve(K, S))